### PR TITLE
fix(Slider): use slider with no linear data

### LINKED ISSUE #7180: Bug - [Slider] : Slider doesn't work with non-liner data
**Describe the problem**
Slider component doesn't work with non-linear data. I've create codesandbox to reproduce issue.
https://codesandbox.io/s/holy-lake-s4fstb?file=/index.js

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/holy-lake-s4fstb?file=/index.js ) demonstrating the problem is appreciated.

1. Open codesandbox 
2. Move slider 
3. It will trough error as attached screenshot

**Expected behavior**
Slider should move without any error with any kind of data (linear or non-linear)

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
image

**What is your environment?**
 - OS: [Linux fedora]
 - Browser [chrome,]
 - Version [ 99.0.4844.84 (Official Build) (64-bit)]

**What is your product and what release date are you targeting?**
Red Hat OpenShift Streams for Apache Kafka (https://console.redhat.com/beta/application-services/streams/kafkas)

**Any other information?**

### PR DESCRIPTION
**Pull request by bartoval**
**What**: Closes #7180 

this is the formula we apply
      ``` percentage = (newPercentage * (max - min)) / 100 + min```;

-  normalize  '**new percentage**' : from 100% to max %
- **+ min**: start offset in case the first value is not 0%

### CODE REVIEW

File: packages/react-core/src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Slider/Slider.tsx:1]: It doesn't look like there are any console calls in this file, is this still needed?

**Review by thatblindgeye — APPROVED** [review]
Works really nicely using the code supplied in the original issue! Only had a quick/nitpicky comment below.

**Review by tlabaj — COMMENTED** [review]
Looks good!  I think it would be beneficial to add an example to show the non linear data.

**Comment by bartoval** [comment]
> Looks good! I think it would be beneficial to add an example to show the non linear data.
RIght!  I copied the example I made on the demo

**Review by mcarrano — APPROVED** [review]
Looks good to me. Thanks @bartoval !

**Review by mattnolting — APPROVED** [review]
LGTM 👍

**Review by tlabaj — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**